src/modules/Elsa.Workflows.Core/Pipelines/WorkflowExecution/WorkflowExecutionMiddlewareExtensions.cs

```diff
@@ -11,16 +12,50 @@ public static class WorkflowExecutionMiddlewareExtensions
     /// <summary>
     /// Installs the specified middleware component into the pipeline being built.
     /// </summary>
-    public static IWorkflowExecutionPipelineBuilder UseMiddleware<TMiddleware>(this IWorkflowExecutionPipelineBuilder pipelineBuilder, params object[] args) where TMiddleware: IWorkflowExecutionMiddleware
+    public static IWorkflowExecutionPipelineBuilder UseMiddleware<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] TMiddleware>(
```

Comment:
We are using DynamicallyAccessedMembers here on all functions, however this is only directly the case in the last function.

Reply:
Indeed. It would seem that this has a viral effect. If we remove the attribute from this method, for example, the compiler will generate the following warning:

<img width="948" alt="image" src="https://github.com/elsa-workflows/elsa-core/assets/938393/faa770f7-53f6-415c-9e3b-f6492dd0cb55">

Adding the attribute solves this warning.